src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -1245,5 +1244,134 @@ internal void SaveAccesTokenCacheItem(MsalAccessTokenCacheItem msalAccessTokenCa
                 }
             }
         }
+
+#if !ANDROID_BUILDTIME && !iOS_BUILDTIME && !WINDOWS_APP_BUILDTIME
+        // todo: where to put this documentation
+        ///// <summary>
+        ///// Extension methods used to subscribe to cache serialization events, and to effectively serialize and deserialize the cache
```

Comment:
Do we want to reconsider the fact that these are extension methods?

Reply:
They are no longer extension methods, it's now on an interface.  So I'm not sure where we want this documentation, or if i should just remove it.

Reply:
I think docs should be on the interface. Implementing public class can just point to the interface via  /// See <see cref="IFoo"/>.

Sadly C# comments do not have @ inheritdoc like Java, although R# seems to do smth about it.

https://stackoverflow.com/questions/342964/comment-inheritance-for-c-sharp-actually-any-language